packages/assets-controllers/src/TokensController.ts

```diff
@@ -277,18 +285,11 @@ export class TokensController extends BaseController<
       'NetworkController:getNetworkClientById',
       selectedNetworkClientId,
     );
-    const { allTokens, allIgnoredTokens, allDetectedTokens } = this.state;
     const { chainId } = selectedNetworkClient.configuration;
     this.#abortController.abort();
     this.#abortController = new AbortController();
     this.#chainId = chainId;
-    const selectedAddress = this.#getSelectedAddress();
-    this.update((state) => {
-      state.tokens = allTokens[chainId]?.[selectedAddress] || [];
-      state.ignoredTokens = allIgnoredTokens[chainId]?.[selectedAddress] || [];
-      state.detectedTokens =
-        allDetectedTokens[chainId]?.[selectedAddress] || [];
-    });
+    // No need to update `tokens`, `ignoredTokens`, or `detectedTokens`.
```

Comment:
Was this left here intentionally because we didn't remove the state properties which are just marked as deprecated ?